pulsar-flink/src/test/java/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchAvroSinkExample.java

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
can you rebase to latest master? in latest master, all examples are put under `examples` directory.

https://github.com/apache/pulsar/tree/master/examples/flink-consumer-source/src/main/scala/org/apache/flink/batch/connectors/pulsar/example

Reply:
I fork pulsar code two days ago, now I add this content. thk